src/index.ts

```diff
@@ -24,6 +24,29 @@ export abstract class AlpineComponent {
     [key: string]: any;
 }
 
+export declare type Alpine = {
+	version: string;
+	pauseMutationObserver: boolean;
+	magicProperties: { [name: string]: function };
+	ignoreFocusedForValueBinding: boolean;
+	pauseMutationObserver: boolean;
+	onComponentInitializeds: Array<any>;
+	onBeforeComponentInitializeds: Array<any>;
+	onComponentInitializeds: (callback: function) => void;
+	onBeforeComponentInitializeds: (callback: function) => void;
+	listenForNewUninitializedComponentsAtRunTime: () => undefined;
+	discoverUninitializedComponents: (
+		callback: (rootEl: HTMLElement) => void,
+		el?: HTMLElement = null
+	) => void;
+	discoverComponents: (callback: (rootEl: HTMLElement) => void) => void;
+	start: () => void;
+	addMagicProperty(name: string, callback: ($el: HTMLElement) => void);
+	clone: (component: any, newEl: HTMLElement) => void;
```

Comment:
Shouldn't the `component` argument use `AlpineComponent`?

Reply:
Oh yes true!